### ISSUE TITLE
CMake now broken on Linux

### ISSUE THREAD

**Issue by certik** [jkriege2/JKQtPlotter#20]
The latest CMake change works on macOS, but not on Linux:
```
$ cmake -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX ..
CMake Warning (dev) in CMakeLists.txt:
  No project() command is present.  The top-level CMakeLists.txt file must
  contain a literal, direct call to the project() command.  Add a line of
  code such as

    project(ProjectName)

  near the top of the file, but after cmake_minimum_required().

  CMake is pretending there is a "project(Project)" command on the first
  line.
This warning is for project developers.  Use -Wno-dev to suppress it.

-- The C compiler identification is GNU 7.4.0
-- The CXX compiler identification is GNU 7.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:13 (include):
  include could not find load file:

    JKQtPlotter_LIB_properties


-- 
-- =============================================================================
-- == JKQTPlotterLib Build Information                                        ==
-- =============================================================================
-- Version:           
-- Using compiler:    GNU 7.4.0
-- Architecture:      x86_64-linux-gnu / x86_64-linux-gnu
-- System Name:       Linux
-- Processor Name:    x86_64
-- Using Qt:          5.12.5
--                      [at /home/certik/miniconda3/envs/qt/lib/cmake/Qt5]
-- Using XITS fonts
-- Installing to:     /home/certik/miniconda3/envs/qt
-- =============================================================================
-- 
-- -- Building JKQTCommonSharedLib
CMake Error at /home/certik/miniconda3/envs/qt/share/cmake-3.15/Modules/WriteBasicConfigVersionFile.cmake:43 (message):
  No VERSION specified for WRITE_BASIC_CONFIG_VERSION_FILE()
Call Stack (most recent call first):
  /home/certik/miniconda3/envs/qt/share/cmake-3.15/Modules/CMakePackageConfigHelpers.cmake:225 (write_basic_config_version_file)
  lib/jkqtcommon/CMakeLists.txt:85 (write_basic_package_version_file)


-- Configuring incomplete, errors occurred!
See also "/home/certik/repos/JKQtPlotter/build/CMakeFiles/CMakeOutput.log".
```

**Comment by certik** [comment]
The commit 92daa4d1a8d7f8552af79598fac9381e7e573c34 works on Linux, but the commit 39c43ef8abe609afa911b419be4d02afa9bd8f30 does not.

**Comment by certik** [comment]
The problem is that for some reason the `${PROJECT_VERSION}` variable is empty.

**Comment by certik** [comment]
Fixed by #21.

**Comment by certik** [comment]
#21 fixes the symptom, but the real fix is provided in #24.
